QIDITechnology/QIDITechnology.ini

```diff
@@ -0,0 +1,3300 @@
+inherits = *QIDI ASA*
+advance_pressure = 0.011
+min_print_speed = 10
+temperature = 260
+volume_temperature = 0
+compatible_printers_condition = printer_model=="X-smart 3" and nozzle_diameter[0]==0.8
+
+[filament:QIDI PA12-CF @X-smart 3 0.8 nozzle]
+inherits = *QIDI PA12-CF*
+compatible_printers_condition = printer_model=="X-smart 3" and nozzle_diameter[0]==0.8
+
+[filament:QIDI PAHT-CF @X-smart 3 0.8 nozzle]
+inherits = *QIDI PAHT-CF*
+compatible_printers_condition = printer_model=="X-smart 3" and nozzle_diameter[0]==0.8
+
+[filament:QIDI PC/ABS-FR @X-smart 3 0.8 nozzle]
+inherits = *QIDI PC/ABS-FR*
+volume_temperature = 0
+compatible_printers_condition = printer_model=="X-smart 3" and nozzle_diameter[0]==0.8
+
+[filament:QIDI PET-CF @X-smart 3 0.8 nozzle]
+inherits = *QIDI PET-CF*
+compatible_printers_condition = printer_model=="X-smart 3" and nozzle_diameter[0]==0.8
+
+[filament:QIDI PETG-Tough @X-smart 3 0.8 nozzle]
+inherits = *QIDI PETG-Tough*
+temperature = 240
+compatible_printers_condition = printer_model=="X-smart 3" and nozzle_diameter[0]==0.8
+
+[filament:QIDI PLA Rapido @X-smart 3 0.8 nozzle]
+inherits = *QIDI PLA Rapido*
+advance_pressure = 0.008
+compatible_printers_condition = printer_model=="X-smart 3" and nozzle_diameter[0]==0.8
+
+[filament:QIDI PLA Rapido Matte @X-smart 3 0.8 nozzle]
+inherits = *QIDI PLA Rapido Matte*
+advance_pressure = 0.008
+compatible_printers_condition = printer_model=="X-smart 3" and nozzle_diameter[0]==0.8
+
+[filament:QIDI PLA-CF @X-smart 3 0.8 nozzle]
+inherits = *QIDI PLA-CF*
+advance_pressure = 0.008
+filament_max_volumetric_speed = 18
+compatible_printers_condition = printer_model=="X-smart 3" and nozzle_diameter[0]==0.8
+
+[filament:QIDI UltraPA @X-smart 3 0.8 nozzle]
+inherits = *QIDI UltraPA*
+compatible_printers_condition = printer_model=="X-smart 3" and nozzle_diameter[0]==0.8
+
+# X-smart 3 0.8 nozzle Generic filament preset
+[filament:Generic ABS @X-smart 3 0.8 nozzle]
+inherits = *Generic ABS*
+advance_pressure = 0.011
+compatible_printers_condition = printer_model=="X-smart 3" and nozzle_diameter[0]==0.8
+
+[filament:Generic PETG @X-smart 3 0.8 nozzle]
+inherits = *Generic PETG*
+temperature = 240
+compatible_printers_condition = printer_model=="X-smart 3" and nozzle_diameter[0]==0.8
+
+[filament:Generic PLA @X-smart 3 0.8 nozzle]
+inherits = *Generic PLA*
+advance_pressure = 0.008
+compatible_printers_condition = printer_model=="X-smart 3" and nozzle_diameter[0]==0.8
+
+[filament:Generic TPU 95A @X-smart 3 0.8 nozzle]
+inherits = *Generic TPU 95A*
+temperature = 220
+compatible_printers_condition = printer_model=="X-smart 3" and nozzle_diameter[0]==0.8
+
+# Common printer preset
+[printer:*common*]
+autoemit_temperature_commands = 1
+auxiliary_fan = 0
+bed_custom_model = 
+bed_custom_texture = 
+bed_exclude_area = 0x0
+bed_shape = 0x0,325x0,325x325,0x325
+before_layer_gcode = 
+between_objects_gcode = 
+binary_gcode = 0
+chamber_fan = 1
+chamber_temperature = 0
```

Comment:
`chamber_temperature` is not valid key for printer section, please remove it from all printer profiles